fixtures/external-durable-objects-app/tests/index.test.ts

```diff
@@ -1,10 +1,9 @@
-import { fork } from "child_process";
 import * as path from "path";
 import { setTimeout } from "timers/promises";
 import { fetch } from "undici";
 import { afterAll, beforeAll, describe, expect, it } from "vitest";
 import { unstable_dev } from "wrangler";
-import type { ChildProcess } from "child_process";
+import { runWranglerPagesDev } from "../../shared/src/run-wrangler-long-lived";
```

Comment:
Refactored this test suite to use runWranglerPagesDev as it was doing pretty much the same thing with fork